core/src/main/java/com/netflix/conductor/core/events/FindUpdateAction.java

```diff
@@ -58,10 +59,17 @@ public FindUpdateAction(WorkflowExecutor executor) {
 		}
 
 		// Lets find WAIT + IN_PROGRESS tasks directly via edao
+		String ndcValue = NDC.peek();
 		boolean taskNamesDefined = CollectionUtils.isNotEmpty(findUpdate.getTaskRefNames());
 		List<Task> tasks = executor.getPendingSystemTasks(Wait.NAME);
 		tasks.parallelStream().forEach(task -> {
+			boolean ndcCleanup = false;
 			try {
+				if (StringUtils.isEmpty(NDC.peek())) {
```

Comment:
These are logging changes. 'tasks.parallelStream()' produces separate treads to process data in parallel but NDC value (owner filed in log entries) not propagated to these threads which results in empty owner field for these messages.